charts/bulk-scan-processor/values.yaml

```diff
@@ -24,10 +24,7 @@ java:
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
     STORAGE_BLOB_PUBLIC_KEY: "nonprod_public_key.der"
     STORAGE_URL: https://bulkscan.{{ .Values.global.environment }}.platform.hmcts.net
-    BULK_SCANNING_DB_USER_NAME: bulk_scanner@bulk-scan-processor-{{ .Values.global.environment }}
-    BULK_SCANNING_DB_NAME: bulk_scan
-    BULK_SCANNING_DB_HOST: bulk-scan-processor-{{ .Values.global.environment }}.postgres.database.azure.com
-    BULK_SCANNING_DB_PORT: "5432"
+    FLYWAY_SKIP_MIGRATIONS: true
     FLYWAY_URL: jdbc:postgresql://bulk-scan-processor-{{ .Values.global.environment }}.postgres.database.azure.com:5432/bulk_scan?sslmode=require
     FLYWAY_USER: bulk_scanner@bulk-scan-processor-{{ .Values.global.environment }}
     FLYWAY_NOOP_STRATEGY: "true"
```

Comment:
oh, actually this one uses some older config called `FLYWAY_NOOP_STRATEGY`. need a change ;)

Reply:
or if you want to - can use this and remove the old one. but then need a change in src code